labelbox/schema/foundry/foundry_client.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, client):
     def _create_app(self, app: App) -> App:
         field_names_str = "\n".join(APP_FIELD_NAMES)
         query_str = f"""
-            mutation CreateDataRowAttachmentPyApi(
+            mutation CreateFoundryApp(
```

Comment:
Should add `PyApi` to the end so we can track it as an SDK invoked GQL request

`CreateFoundryAppPyApi`

Reply:
thanks for the tip, Kevin!